src/content/add-to-app/ios/add-flutter-screen.md

```diff
@@ -49,29 +49,29 @@ into a `ContentView` using the `environment()` view modifier.
 
  ```swift title="MyApp.swift"
 import SwiftUI
-import Flutter
-// The following library connects plugins with iOS platform code to this app.
-import FlutterPluginRegistrant
-
-@Observable
-class FlutterDependencies {
-  let flutterEngine = FlutterEngine(name: "my flutter engine")
-  init() {
-    // Runs the default Dart entrypoint with a default Flutter route.
-    flutterEngine.run()
-    // Connects plugins with iOS platform code to this app.
-    GeneratedPluginRegistrant.register(with: self.flutterEngine);
-  }
-}
++ import Flutter
++ // The following library connects plugins with iOS platform code to this app.
++ import FlutterPluginRegistrant
+
++ @Observable
++ class FlutterDependencies {
++   let flutterEngine = FlutterEngine(name: "my flutter engine")
++   init() {
++     // Runs the default Dart entrypoint with a default Flutter route.
++     flutterEngine.run()
++     // Connects plugins with iOS platform code to this app.
++     GeneratedPluginRegistrant.register(with: self.flutterEngine);
++   }
++ }
```

Comment:
Let's see how this diff (+/-) looks like on the website.

Reply:
The +/- doesn't look good because: 

- No highlight like github page. 
- Almost most of the code are marked with `+`

I'm gonna revert. @loic-sharma

Reply:
Hm bummer...

Here was my feedback at the time:

> This step tells you to modify your entrypoint `MyApp.swift` file. About half the content is the normal app boilerplate, and the other half is Flutter additions. It is easy to miss additions.
>
> For example, the engine is injected using the `environment()` view modifier. This is nested in the `Scene` declaration. Forgetting to add this won’t result in an error until subsequent steps.
>
> 💡 Suggestion: Consider making this code sample a diff so that it’s clear which lines are added.

Perhaps we should split this content into multiple steps so that what should be added is clearer?